website/content/docs/concepts/domain-model/index.mdx

```diff
@@ -51,7 +51,26 @@ The highest-level in Boundary utilizes the concepts of [Roles](/docs/concepts/do
 
 ## Target and Host Resources
 
-Boundary exposes endpoints to users as [Targets](/docs/concepts/domain-model/targets). A target is a resource representing a networked service, with an associated set of permissions, that allows a user to connect and interact with Boundary within a single  session. A target may contain references to [Host](/docs/concepts/domain-model/hosts) sources and [Credential](/docs/concepts/domain-model/credentials) sources. Users who access a target create an authorized session to one of its host sources with returned credentials from the target’s credential source.
+Boundary exposes endpoints to users as [Targets](/docs/concepts/domain-model/targets). A target is a resource representing a networked service, with an associated set of permissions, that allows a user to connect and interact with Boundary within a single session. A target may contain references to [Host](/docs/concepts/domain-model/hosts) sources or an address, and [Credential](/docs/concepts/domain-model/credentials) sources. Users who access a target create an authorized session to its address or one of its host sources with returned credentials from the target’s credential source.
+
+A target exposes two distinct and mutually exclusive configuration paths for its
+representation of a networked service:
+
+- Target with an address
+- Target with host sources
+
+A target with an address represents a target that is set up with a single IP
+address or DNS name directly on the target resource. This mechanism provides
+benefits for new Boundary users and users who don't require Boundary's
+flexibility and scalability, as it's less complex to set up and understand.
```

Comment:
I think the phrasing here seems a little odd. Mainly the "don't require Boundary's flexibility and scalability". It isn't "boundary's" flexibility and scalability that is being avoided here, using an Address on a Target is still part of the boundary domain model. Instead it is avoiding the need for Hosts, Host Sets, Host Catalogs. So maybe something like:


```suggestion
address or DNS name directly on the target resource. This mechanism provides
benefits for new Boundary users and users who do not need the flexibility
provided by Hosts, Host Sets, and Host Catalogs.
It is well suited for cases where there are stand-alone hosts
that should not be grouped together and treated as equal from an access control point of view.
```

Reply:
Good point - Makes it much clearer, thanks a lot!

Reply:
Addressed: https://github.com/hashicorp/boundary/commit/cf145792f9ae1161ddc9521953f39ccabd116bab